src/binary/timestamp.rs

```diff
@@ -11,75 +11,89 @@ use crate::{
     types::timestamp::{Precision, Timestamp},
 };
 
-const ENCODED_BUFFER_SIZE: usize = 16;
+const MAX_TIMESTAMP_LENGTH: usize = 16;
 
 /// Extends [`Timestamp`] to support [Ion binary].
```

Comment:
Minor doc comment suggestion:

```suggestion
/// Provides support to write [`Timestamp`] into [Ion binary].
```